tests/nvidianetwork/deploy-nno-test.go

```
@@ -355,24 +359,8 @@ var _ = Describe("NNO", Ordered, Label(tsparams.LabelSuite), func() {
-				_ = nfd.DeleteNFDCSV(inittools.APIClient)
-
-				By("Delete NFD Subscription in NFD namespace")
-				_ = nfd.DeleteNFDSubscription(inittools.APIClient)
-
-				By("Delete NFD OperatorGroup in NFD namespace")
-				_ = nfd.DeleteNFDOperatorGroup(inittools.APIClient)
-
-				By("Delete NFD Namespace in NFD namespace")
-				_ = nfd.DeleteNFDNamespace(inittools.APIClient)
+			if nfdInstance.CleanupAfterInstall && cleanupAfterTest {
+				nfd.Cleanup(inittools.APIClient)
```

golangci-lint (1.64.8)

Error return value of `nfd.Cleanup` is not checked

(errcheck)